src/README.md

```diff
@@ -112,25 +114,55 @@ Compiled binaries may be flashed onto the EFR32 using various tools from the [J-
 
 ### Simplicity Commander
 
-Simplicity Commander provides a graphical interface for J-Link Commander. It's included as part of [Simplicity Studio][simplicity-studio] and is also available as a standalone application
+Simplicity Commander provides command line and graphical interfaces for J-Link Commander. It's included as part of [Simplicity Studio][simplicity-studio] and is also available as a standalone application
 
 #### Download Links
 
 - [Linux](https://www.silabs.com/documents/public/software/SimplicityCommander-Linux.zip)
 - [Mac](https://www.silabs.com/documents/public/software/SimplicityCommander-Mac.zip)
 - [Windows](https://www.silabs.com/documents/public/software/SimplicityCommander-Windows.zip)
 
+For Windows and Linux, the Simplicity Commander executable can be found in the location that the package is extracted.  
+On MacOS, the executable can be found at the following directory:
+
 ```bash
-$ <path-to-simplicity-studio>/developer/adapter_packs/commander/commander
+$ /Applications/Commander.app/Contents/MacOS
 ```
 
+#### **Commander UI**
+
+Launch the Simplicity Commander executable "commander" at the path stated above to open the Commander UI, and perform the following steps to flash build artifacts to a target device:
+
 1. In the J-Link Device drop-down list select the serial number of the device to flash.
 2. Click the Adapter Connect button.
 3. Ensure the Debug Interface drop-down list is set to SWD and click the Target Connect button.
 4. Click on the Flash icon on the left side of the window to switch to the flash page.
 5. In the Flash MCU pane, enter the path of the `ot-cli-ftd.s37` file or choose the file with the Browse... button.
 6. Click the Flash button located under the Browse... button.
 
+#### **Commander CLI**
+
+The Simplicity Commander CLI provides another method of flashing binaries to a device without invoking an interactive UI.  
+When using Simplicity Commander in CLI mode, it is highly reccomended to add the directory of the executable to your environment's PATH. This can be done with the following command:
```

Comment:
```suggestion
When using Simplicity Commander in CLI mode, it is highly recommended to add the directory of the executable to your environment's `PATH` variable. This can be done with the following command:
```